azure-pipelines.yml

```diff
@@ -0,0 +1,32 @@
+trigger:
+  branches:
+    include: ['*']
+  tags:
+    include: ['*']
+
+jobs:
+  - job: Test
+  pool:
+    vmImage: 'Ubuntu-18.04'
+    vmImage: 'macOS-10.13'
+    vmImage: 'vs2017-win2016'
+  strategy:
+    matrix:
+      Python36:
+        python.version: '2.7'
+      Python37:
+        python.version: '3.6'
+    maxParallel: 2
+    
+    displayName: Performing test 
+    steps:
+      - task: UsePythonVersion@0
+      inputs:
+      versionSpec: '$(python.version)'
+      - script: ./configure
+      - script: ./bin/py.test -vvs tests/commoncode
+      - task: PublishTestResults@2
```

Comment:
What is this doing?

Reply:
Publish the result like this https://docs.microsoft.com/en-us/azure/devops/pipelines/tasks/test/publish-code-coverage-results?view=azure-devops